addons/account_bank_statement_import/account_bank_statement_import.py

```diff
@@ -25,7 +25,11 @@ class AccountBankStatementImport(models.TransientModel):
     _name = 'account.bank.statement.import'
     _description = 'Import Bank Statement'
 
+    def _default_options(self):
+        return []
+
     data_file = fields.Binary(string='Bank Statement File', required=True, help='Get you bank statements in electronic format from your bank and select them here.')
+    import_type = fields.Selection(_default_options, string="Import File Type", required=True, help='Select bank statement format to import.')
```

Comment:
why need to call default_options function. ? selection value is static and directly we can override selection field attribute in new api using "select_add" attribute.